kcidb/templates/build.j2

```diff
@@ -24,6 +24,43 @@
     -}}
 {% endmacro %}
 
+{%- macro status(architecture,
+                invalid_build_count,
+                valid_build_count,
+                unknown_build_count,
+                build_fmt) -%}
```

Comment:
I think this should be called `architecture_status`, as in "producing a summary for builds in an "architecture" container". However, it might turn out that we don't need this particular separate macro after all.